content/manuals/build/building/variables.md

```diff
@@ -507,6 +509,23 @@ Usage:
 $ export BUILDX_CONFIG=/usr/local/etc
 ```
 
+### BUILDX_CPU_PROFILE
+
+{{< introduced buildx 0.18.0 >}}
+
+If specified, Buildx generates a `pprof` CPU profile at the specified location.
+
+A CPU profile records where your program spends CPU time. By analyzing a CPU
+profile, you can identify functions that consume the most processing power.
+This helps you optimize code paths, reduce latency, and improve overall
+performance.
```

Comment:
Should we maybe make a note that this environment variable is only really helpful for development of buildx itself? This passage makes it sound like this can help you profile your builds when this is a profile of buildx itself. It might be useful to document just so that it's documented, but we shouldn't make it sound like this environment variable is particularly helpful to the average person.

Reply:
thanks, yes I wasn't really sure if this data is somehow useful for building. I wrote this down as a "better safe than sorry" measure. I'll incorporate this.